services/brig/test/integration/API/User/Auth.hs

```diff
@@ -759,9 +805,8 @@ testMissingCookie z b = do
     !!! errResponse
   where
     errResponse = do
-      const 403 === statusCode
-      const (Just "Missing cookie") =~= responseBody
-      const (Just "invalid-credentials") =~= responseBody
+      const 400 === statusCode
+      const (Just "Header Cookie is required") =~= responseBody
```

Comment:
Error codes and labels are now unchanged. This required a non-trivial amount of fiddling with the way errors are generated, but it should all be compatible now. As discussed, the behaviour in terms of error messages is still not 100% the same in case a single request contains multiple errors (such as unparseable body and missing or unparseable access token at the same time).